### DISCUSSION TITLE
Building shopping cart to learn state management

### DISCUSSION THREAD

**Discussion by anish-u** [notJust-dev/notjustdev#165] [Ideas]
# Building shopping cart to learn state management

We could make this feature using different state management libraries like
- Redux
- Zustand
- Recoil
- Jotai

## Simple wireframe

![image](https://github.com/notJust-dev/DEVember/assets/71644254/7aafde5f-6054-48aa-a628-2ac62cde4105)

**Comment by josephmyalla** [comment]
The exact same app is available in his youtube series front end and backend https://www.youtube.com/watch?v=Gvxv6JCabA8&t=10s
